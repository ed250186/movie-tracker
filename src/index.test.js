import React from 'react';
import {shallow} from 'enzyme';
import {root} from './index';


describe('src index', () => {

  xit('should match snapshot', () => {
    let wrapper = shallow(<root/>)
    expect(wrapper).toMatchSnapshot()
  })

})