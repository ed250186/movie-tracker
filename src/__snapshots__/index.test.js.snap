// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src Root should match snapshot 1`] = `
<Provider>
  <BrowserRouter>
    <Route
      component={[Function]}
      path="/"
    />
  </BrowserRouter>
</Provider>
`;
